packages/backend/src/db/storage/update-mailing-lists.ts

```diff
@@ -28,11 +28,65 @@ export const UPDATE_MAILING_LISTS_TEMPLATE: Template = {
       sampleUrl:
         'https://plumber.gov.sg/tiles/ba2150f6-14d5-44cf-8a77-083c18f43518/c6b75dfa-9fa9-494c-b027-773da38ebaff',
       sampleUrlDescription: TILES_SAMPLE_URL_DESCRIPTION,
+      parameters: {
+        filters: [
+          {
+            columnId: 'Name',
+            value: 'Anna Lee',
+            operator: 'equals',
+          },
+        ],
+        returnLastRow: true,
+      },
     },
     {
       position: 3,
       appKey: 'tiles',
       eventKey: 'updateSingleRow',
+      parameters: {
+        rowId: '{{Replace with row id result from step 2}}',
+        rowData: [
+          {
+            columnId: 'Email',
```

Comment:
As discussed, those parameters to be replaced should be wrapped with {{ }}. So for this columnId, it should take the form of e.g. {{TILE_COL_ID|Email}}

Reply:
basically `{{ }}` tells u it is to be replaced, 
TILE_COL_ID, tells u what replace function to use
and Email is the arg

Reply:
hmm ok actually i realised that i could just create a `replacementMap` for every placeholder since they do similar things e.g. `user_email: user.email` and then just replace them in a function, feel free to tell me if its clean enough thanks